### ISSUE TITLE
Quest 862 not showing on map

### ISSUE THREAD

**Issue by anon1231823** [Questie/Questie#6746]
### Description

The cooking quest 862 in the barrens has no market even though it's available
![image](https://github.com/user-attachments/assets/1cd0bcb0-411a-42db-a4cc-bd207bf0aace)

### Questie version

v10.10.2

### Game flavor

Era/HC/Anniversary

**Comment by Cabro** [comment]
That's not something we can fix atm. maybe @BreakBB wants to fix the spell requirements?

**Comment by Cabro** [comment]
The quest requires Journeyman. We can add the spell ID for journeyman, but when you train next rank on your profession, you wont have it anymore. And it won't show if you are higher than journeyman.

Solution is to have requiredspell take multiple values, and change a bit of logic, or add new field for requiredminimumskilllevel or something.

**Comment by Cabro** [comment]
It's not a lie, that spell acts like all the other previous profession ranks. But you lose this one when you train next one.

**Comment by BreakBB** [comment]
But it returns false when a higher rank is available from what I see in your example?

**Comment by Cabro** [comment]
Correct. :D Only blizzard knows whatever logic they use internally for that

**Comment by anon1231823** [comment]
#1756 also references this quest
